models/movement.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const movementSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    require: true
+  },
+  description: {
+    type: String,
+    validate: {
+      validator: function(v) {
+        return v.length <= 200;
+      },
+      message: '{VALUE} should not be longer than 200 characters!'
+    }
```

Comment:
Nice! But you could also have just used the built-in String validator `maxlength`.